### PR TITLE
First pass at more objects, flattening and run3 definitions

### PR DESCRIPTION
**Pull request by cmantill**
Nothing final yet but more developments (including 2022 jet veto maps)

### CODE REVIEW

File: src/HH4b/processors/bbbbSkimmer.py [hunk 1]
```diff
@@ -390,31 +398,38 @@ def process(self, events: ak.Array):
...
         shift = ptlabel + mlabel
 
         dijetVars[f"DijetPt{shift}"] = Dijet.pt
         dijetVars[f"DijetMass{shift}"] = Dijet.M
         dijetVars[f"DijetEta{shift}"] = Dijet.eta
 
-        # dijetVars[f"bbFatJetPtOverDijetPt{shift}"] = bbJet.pt / Dijet.pt
-        # dijetVars[f"VVFatJetPtOverDijetPt{shift}"] = VVJet.pt / Dijet.pt
-        # dijetVars[f"VVFatJetPtOverbbFatJetPt{shift}"] = VVJet.pt / bbJet.pt
+        if shift == "":
```
Comment (rkansal47) [inline src/HH4b/processors/bbbbSkimmer.py:439]: Aren't these also affected by JECs?

File: src/HH4b/processors/bbbbSkimmer.py [hunk 2]
```diff
@@ -166,133 +172,134 @@ def process(self, events: ak.Array):
...
+                        events.HLT[trigger]
+                        for trigger in self.HLTs[year]
+                        if trigger in events.HLT.fields
+                    ]
                 ),
                 axis=0,
             )
             add_selection("trigger", HLT_triggered, *selection_args)
 
-        # pt, eta cuts: check if jet passes pT cut in any of the JEC variations
```
Comment (rkansal47) [inline src/HH4b/processors/bbbbSkimmer.py:267]: Are you removing the fatjet pT preselection?

**Review by rkansal47 — COMMENTED** [review]
LGTM, just confused about two things.

File: src/HH4b/processors/bbbbSkimmer.py (hunk 1 above)
Comment (cmantill) [reply src/HH4b/processors/bbbbSkimmer.py:439]: ok added those

File: src/HH4b/processors/bbbbSkimmer.py (hunk 2 above)
Comment (cmantill) [reply src/HH4b/processors/bbbbSkimmer.py:267]: oops put it back - we should consider making another processor for the analysis that does not require fatjets

Comment (rkansal47) [reply src/HH4b/processors/bbbbSkimmer.py:267]: Oh right, yeah I didn't consider the different pre-selections for different analyses. Probably better then to modularize the object selection, pre-selection, and skimming, and add an option for which analysis is being run. (For future)

**Pull request merged**